src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventFiltersInterface/toEventQueryCriteriaConverter/optionSet.js

```diff
@@ -11,9 +9,7 @@ import type {
     OptionSetFilterData,
 } from '../../../../EventsList/eventList.types';
 
-const stringifyNumber = (rawValue: number) => {
-    rawValue.toString();
-};
+const stringifyNumber = (rawValue: number) => rawValue.toString();
```

Comment:
Looks like this was not returning a value before. Perhaps that was intended but I doubt it. Can you also check Joakim that I am not breaking anything here?